students/scott/Coding_Bat/list_1.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+# first_last6
+
+def first_last6(nums):
+    return nums[0] == 6 or nums[-1] == 6
+
+assert first_last6([1, 2, 6]) == True
```

Comment:
when testing for somethign being True or FAlse, better to use is:

```
assert first_last6([1, 2, 6])  is True
```
`is` should be used only for the singletons: True False, None.